### ISSUE TITLE
Directories end with `/` with `markDirectories: false` and `//` with `markDirectories: true`

### ISSUE THREAD

**Issue by pvdlg** [mrmlnc/fast-glob#46]
With the following directory tree:
```
+-- test/
|   +-- dir
|       +-- file.js
```

```js
fg.sync('test/**', {onlyFiles: false, markDirectories: false});

// Result => ['test/dir/', 'test/dir/file.js']
// Should be => ['test/dir', 'test/dir/file.js']
```
```js
fg.sync('test/**', {onlyFiles: false, markDirectories: true});

// Result => ['test/dir//', 'test/dir/file.js']
// Should be => ['test/dir/', 'test/dir/file.js']
```

**Comment by mrmlnc** [comment]
Thanks for feedback! 🌮 

I really appreciate it! Fixed by #43.

**Comment by pvdlg** [comment]
Thanks so much!

I trying to migrate [globby](https://github.com/sindresorhus/globby/pull/68) from `node-glob` to `fast-glob` to benefit from the perf improvements.
So I'm doing quite some tests :)

`fast-glob` is awesome! The perf are really good, and the node ecosystem really need a well maintained glob solution.
